src/compile/axis/properties.ts

```diff
@@ -11,7 +11,7 @@ import {NOMINAL, ORDINAL, Type} from '../../type';
 import {contains, normalizeAngle} from '../../util';
 import {isSignalRef} from '../../vega.schema';
 import {UnitModel} from '../unit';
-import {getAxisConfig} from './config';
+import {AxisConfigs, getAxisConfig} from './config';
 
 // TODO: we need to refactor this method after we take care of config refactoring
```

Comment:
Still relevant

Reply:
will remove in a subsequent refactor PR, so we don't have to rebase this one.